disnake/guild.py

```diff
@@ -222,6 +222,7 @@ class Guild(Hashable):
         - ``HUB``: Guild is a student hub.
         - ``INVITE_SPLASH``: Guild's invite page can have a special splash.
         - ``LINKED_TO_HUB``: Guild is linked to a student hub.
+        - ``MEMBER_PROFILES``: Guild members can set per-guild profiles if they have nitro.
```

Comment:
this isn't documented, and doesn't seem to do anything

Reply:
https://github.com/DisnakeDev/disnake/pull/694/commits/8d2adfa6b5c8381bc9bf1bad4638d9fd1134d409